### PR TITLE
Fixing Puppetfile typo

### PR DESCRIPTION
**Pull request by shapi78**

### CODE REVIEW

File: Puppetfile [hunk 1]
```diff
@@ -28,7 +28,8 @@ mod 'puppetlabs/catalog_preview', :latest
 mod 'puppet/archive', :latest
 mod 'puppetlabs/inifile', :latest
 mod 'graduway/ec2tagfacts',
-  :git => 'https://github.com/graduway/ec2tagfacts',
+  :git => 'https://github.com/graduway/ec2tagfacts', 
+  :branch => 'master'
```
Comment (antgel) [inline Puppetfile]: `master` is the default according to https://puppet.com/docs/pe/2017.3/code_management/puppetfile.html#declare-content-from-a-relative-control-repo-branch, please remove.

File: Puppetfile [hunk 2]
```diff
@@ -60,8 +61,7 @@ mod 'puppetlabs/dummy_service', :latest
 # mod 'puppetlabs/firewall', :latest
 
 # Used by psick::vpn::openvpn
-# mod 'luxflux/openvpn',
-#  :git => 'https://github.com/luxflux/puppet-openvpn'
+mod 'puppet-openvpn', '7.1.0'
```
Comment (antgel) [inline Puppetfile:63]: Shouldn't this be `luxflux/puppet-openvpn`?

**Review by antgel — CHANGES_REQUESTED** [review]
Please amend the existing commit rather than adding another one.

File: Puppetfile (hunk 2 above)
Comment (shapi78) [reply Puppetfile:63]: No, we use this module, it provides more flexability

**Review by antgel — APPROVED** [review]

**Pull request merged**